core-primitives/node-api/api-client-extensions/src/encointer.rs

```diff
@@ -17,13 +17,15 @@
 
 use crate::ApiResult;
 use encointer_primitives::{
-	communities::{CommunityIdentifier, GeoHash},
-	scheduler::CeremonyIndexType,
```

Comment:
I think this encointer-specific stuff should not be in the core-primitives stuff, see #35. However, this is an issue for later.

Reply:
Yes, that makes sense